9.mAP/notebooks/0.generate_map_scores_class_level.ipynb

```diff
@@ -0,0 +1,954 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #7.    df_feats = np.array(df_feats)
</pre><p>Confirming: Roshan's code returns both a pd.DataFrame for metadata and np.array for feature values. Is there a specific reason for using 'df_feats'  in <code>np.array(df_feats)</code>? I want to ensure I'm interpreting this correctly.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/WayScience/pyroptosis_signature_data_analysis/pull/112/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

Reply:
<p>I was getting an error so I converted to numpy array...interesting that I got the error</p>